server/src/test/java/org/apache/iotdb/db/utils/datastructure/LongTVListTest.java

```diff
@@ -30,7 +32,7 @@
 
 
   @Test
-  public void testLongTVList1() {
+  public void testLongTVList1() throws WriteProcessException {
```

Comment:
no need to throw `WriteProcessException` in this test : )

Reply:
Fixed~